go/database/mpt/verification_test.go

```diff
@@ -43,7 +44,7 @@ var forestFiles = []string{
 
 func TestVerification_VerifyValidForest(t *testing.T) {
 	runVerificationTest(t, func(t *testing.T, dir string, config MptConfig, roots []Root) {
-		if err := VerifyFileForest(dir, config, roots, NilVerificationObserver{}); err != nil {
+		if err := VerifyMptState(dir, config, roots, NilVerificationObserver{}); err != nil {
```

Comment:
I do not think these tests should be all changed to test one layer above the forest when they actually test still the same as they used to test before.

Reply:
I wanted to include the whole verification but I guess you are right. Should we the tests be renamed to `TestForestVerification...` since we have now multiple verifications and they only test one part of it?